src/sentry/sentry_apps/api/serializers/request.py

```diff
@@ -17,7 +17,9 @@ class RequestSerializer(Serializer):
     def __init__(self, sentry_app: SentryApp) -> None:
         self.sentry_app = sentry_app
 
-    def get_attrs(self, item_list: list[Any], user: Any, **kwargs: Any) -> MutableMapping[Any, Any]:
+    def get_attrs(
```

Comment:
Neat, we had a response serializer in the rest_framework serializer directory 🫠

Reply:
:bufo-melting: